assets/options.html

```diff
@@ -2,6 +2,7 @@
   <head>
     <meta charset="UTF-8" />
     <title></title>
+    <link rel="stylesheet" href="chrome://global/skin/in-content/common.css">
```

Comment:
Do you know if this is supposed to work in MS Edge as well? The readme of [webext-base-css](https://github.com/fregante/webext-base-css) mentions just Firefox. I'm asking because it looks like they added their own protocol `edge://`

Reply:
Indeed, this only loads Firefox’ style since Firefox effectively doesn't support `options.chrome_style`.

I don't know how it looks in Edge, but I assume it just matches Chrome.